Impossible-Agent-Suite/gas-control-agent/src/agent.ts

```diff
@@ -1,27 +1,36 @@
-import BigNumber from 'bignumber.js';
 import { Finding, HandleTransaction, TransactionEvent } from 'forta-agent';
-import { CONTRACTS, createFinding, hexToNumber } from './utils';
+import utils, { AddressValidator } from './utils';
+import { BigNumber } from 'ethers';
+
+const THRESHOLD: BigNumber = BigNumber.from(10); // gas price
+const DECIMALS: BigNumber = BigNumber.from(10).pow(9);
```

Comment:
I think you can directly use `GAS_DECIMALS` returned in utils to avoid declarations redundancy. Otherwise, `GAS_DECIMALS` is never used